modal/_functions.py

```diff
@@ -473,51 +473,41 @@ def from_local(
 
         explicit_mounts = mounts
 
-        if is_local():
```

Comment:
This is just an unindent, with the `is_local()` getting moved to the inner auto mounts `if` condition